### ISSUE TITLE
Add honggfuzz coverage-based minimizer.

### ISSUE THREAD

**Issue by pventuzelo** [rust-fuzz/honggfuzz-rs#26]
@robertswiecki have implemented a coverage-based minimizer in honggfuzz recently.

https://github.com/google/honggfuzz/issues/195#issuecomment-538805274

It will be awesome to have this feature available over cargo like:
`cargo hfuzz cmin ...`

**Comment by Byter09** [comment]
I just looked at the [usage](https://github.com/google/honggfuzz/blob/master/docs/USAGE.md) and it has a `--minimize` option, but the packed honggfuzz version in this crate is not up-to-date yet. Would this suffice @pventuzelo? If so, should be a simple fix.

**Comment by pventuzelo** [comment]
Yes i think it should work well just by providing this option flag ;) 
Thx @Byter09

**Comment by Byter09** [comment]
This is making progress in form of https://github.com/google/honggfuzz/pull/379 and https://github.com/google/honggfuzz/pull/380. Can't wait for that to land in here. :partying_face:

**Comment by drahnr** [comment]
I'll take a stab at it once #54 is merged and #52 is concluded.
